includes/blocks/class-sensei-lesson-blocks.php

```diff
@@ -87,7 +87,7 @@ public function initialize_blocks() {
 		 *
 		 * @return {string[][]} Array of blocks to use as the default initial state for a lesson.
 		 */
-		$post_type_object->template = apply_filters( 'sensei_lesson_block_template', $block_template, $post_type_object->template );
+		$post_type_object->template = apply_filters( 'sensei_lesson_block_template', $block_template, $post_type_object->template ?? '' );
```

Comment:
Does it make more sense to use an empty array here instead of an empty string since the parameter is a `string[][]`?

Reply:
Updated in 2571f8b